tests/friendship_pb2.py

```diff
@@ -0,0 +1,377 @@
+# -*- coding: utf-8 -*-
```

Comment:
The generated files (this one and the other one) are missing the Apache license header, so RAT check fails. I suggest adding it to rat ignores; grpc compiler might be able to add the header, but that is probably needless complication to worry about, all the time.

Reply:
> This is _very_ end-to-end test!

Indeed! I would say, now that with http2/tcp adaptors, we are going to see
more and more of these kind of tests. Any suggestion on re-structuring or so?

Reply:
I will verify if protoc allows adding the disclaimer on top of each generated file.
Otherwise I will check how to skip this validation.

Reply:
validation skipped